components/integrations/bots/bots.jsx

```diff
@@ -146,14 +147,14 @@ export default class Bots extends React.PureComponent {
                     key={bot.user_id}
                     bot={bot}
                     owner={this.props.owners[bot.user_id]}
-                    accessTokens={this.props.accessTokens[bot.user_id] || {}}
+                    accessTokens={this.props.accessTokens[bot.user_id] ? {[bot.user_id]: this.props.accessTokens[bot.user_id]} : {}}
```

Comment:
This is not exactly the same, and I don't understand why this change is needed here.

Reply:
Ok, so the situation is as follows:
I believe I found a bug in Bots => Bot communication.
Bot component expects to receive accessTokens in shape of `{[botid]:{id:'tokenid',...}}`.
But the Bots component didn't pass the parameter correctly, it passed just `{id:'tokenid',...}` thus breaking the `Object.values` loop inside Bot.jsx

This change passes the parameter in the expected shape

Reply:
I think you are wrong here. I think Bot.jsx is expecting a object of `{accessTokenId: accessToken}`, the `accessToken` property is an object of `{userId: {accessTokenId: accessToken}}`. And probably this also explain the problem with the `token.id` check that @crspeller commented